OrbitGl/App.cpp

```diff
@@ -597,6 +597,13 @@ void OrbitApp::StopCapture() {
 void OrbitApp::ClearCapture() {
   Capture::ClearCaptureData();
 
+  // Trigger deallocation of previous sampling related data.
+  sampling_report_ = nullptr;
```

Comment:
Nit: This should be handled in the subsequent call to OrbitApp::AddSamplingReport, since sampling_report_ is reassigned there. Is there a reason to do this explicit nullptr assignment here?

Reply:
Done, thanks!